presto-kudu/src/test/java/io/prestosql/plugin/kudu/KuduQueryRunnerFactory.java

```diff
@@ -97,6 +97,38 @@ public static QueryRunner createKuduQueryRunnerTpch(TestingKuduServer kuduServer
         }
     }
 
+    public static QueryRunner createKuduQueryRunnerTpch(
```

Comment:
We can modify `QueryRunner createKuduQueryRunnerTpch(TestingKuduServer kuduServer, Optional<String> kuduSchemaEmulationPrefix, Iterable<TpchTable<?>> tables)` to re-use this method

Reply:
Modified.